trunk/SUAVE/Methods/Flight_Dynamics/Dynamic_Stability/compute_aero_derivatives.py

```diff
@@ -0,0 +1,134 @@
+## @ingroup Methods-Flight_Dynamics-Dynamic_Stability
```

Comment:
This is static stability not dynamic stability. Change to the other folder

Reply:
changed